src/Tonic/Generator.h

```diff
@@ -76,6 +83,17 @@ namespace Tonic {
       obj->tick(frames, context);
     }
 
+
+  //! Shortcut for wrapping with a Panner. Currently a MonoToStereoPanner but this could/should
+  // be replaced with a better panner that can handle mono or stereo.
+  Generator pan(ControlGenerator pan);
+
+  //! Shortcut for wrapping with a Panner. Currently a MonoToStereoPanner but this could/should
+  // be replaced with a better panner that can handle mono or stereo.
+  Generator pan(float pan);
+
+  void ____recordAdditionAsParameter____(){ obj->hasBeenAdded = true; }
```

Comment:
C++ reserves identifiers with two or more underscores in a row:
http://stackoverflow.com/a/228797/1959975